crypto/src/main/java/org/web3j/crypto/MnemonicUtils.java

```diff
@@ -1,16 +1,17 @@
 package org.web3j.crypto;
 
-import java.net.URL;
-import java.nio.file.Files;
-import java.nio.file.Paths;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-
 import org.bouncycastle.crypto.digests.SHA512Digest;
 import org.bouncycastle.crypto.generators.PKCS5S2ParametersGenerator;
 import org.bouncycastle.crypto.params.KeyParameter;
 
+import java.io.BufferedReader;
```

Comment:
Please can you keep this imports at the top?

Reply:
I'm sorry, the IDE changed the order. Is it now OK?